### DISCUSSION TITLE
Allows users to customize the font color of code suggestions.

### DISCUSSION THREAD

**Discussion by heisen-li** [community/community#11529] [Copilot Conversations]
Change the font size and color of the recommended code based on the theme or environment of the user. It doesn't seem to be possible at the moment.

**Comment by jansorg** [comment]
Which editor or IDE are you using?

**Reply by heisen-li** [reply]
vscode

**Comment by guisaldanha** [comment]
This could be easily solved if we knew which scope to add the desired color in settings.json using the option
     `"editor.tokenColorCustomizations": {
         "textMateRules": [
             {
                  "scope": "scope.from.copilot",
                  "settings": {
                      "foreground": "#cccccc"
                  }
              },
         ]
     },`

**Reply by guisaldanha** [reply]
"workbench.colorCustomizations": {
        "editorGhostText.border": "#ff0000",
        "editorGhostText.foreground": "#ff0000",
}

As @Krzysztof-Cieslak reported in this discussion https://github.com/github/feedback/discussions/7447
